models/turbine_models/custom_models/sdxl_inference/sdxl_compiled_pipeline.py

```diff
@@ -46,6 +47,109 @@
     "full_pipeline": None,
 }
 
+EMPTY_FLAGS = {
+    "clip": None,
+    "unet": None,
+    "vae": None,
+    "pipeline": None,
+}
+
+sdxl_pipeline_bench_f16 = """
+module @sdxl_compiled_pipeline {
```

Comment:
Nit: may be cleaner to move mlir like this to a separate file

Reply:
these, in current state, are way too inflexible to use in production. I can move them while we work on a better solution. In any case, hardcoding stringified .mlir and reading it into iree-compile feels _really_ bad.